src/utils/mesh_client.ts

```diff
@@ -1,87 +1,41 @@
-import {
-    RejectedCode,
-    RejectedKind,
-    RejectedOrderInfo,
-    SignedOrder,
-    ValidationResults,
-    WSClient,
-    WSOpts,
-} from '@0x/mesh-rpc-client';
-import { orderHashUtils } from '@0x/order-utils';
+import { FilterKind, MeshGraphQLClient, OrderWithMetadata } from '@0x/mesh-graphql-client';
 import * as _ from 'lodash';
 
-import { MESH_ORDERS_BATCH_HTTP_BYTE_LENGTH, MESH_ORDERS_BATCH_SIZE } from '../constants';
-import { logger } from '../logger';
+import { MESH_GET_ORDERS_DEFAULT_PAGE_SIZE as DEFAULT_PAGE_SIZE } from '../config';
 
-import { retryableAxios } from './axios_utils';
-import { utils } from './utils';
+export class MeshClient extends MeshGraphQLClient {
+    constructor(public readonly webSocketUrl: string, public readonly httpUrl?: string) {
+        super({
+            webSocketUrl: `${webSocketUrl}/graphql`,
+            httpUrl: httpUrl ? `${httpUrl}/graphql` : undefined,
+        });
+    }
 
-export class MeshClient extends WSClient {
-    public async addOrdersAsync(orders: SignedOrder[], pinned: boolean = false): Promise<ValidationResults> {
```

Comment:
Cool, looks like the graphql client dependency replaces this method?

Reply:
Yep, it's now part of the Mesh GraphQL client 👍